policyengine_us/variables/gov/states/mo/tax/income/deductions/mo_pension_and_ss_or_ssd_deduction_section_c.py

```diff
@@ -30,7 +30,12 @@ def formula(person, period, parameters):
         taxable_social_security_benefits = person(
             "taxable_social_security", period
         )
+        tax_unit_taxable_social_security_benefits = tax_unit.sum(
```

Comment:
This can replace the above code
```suggestion
        tax_unit_taxable_social_security_benefits = add(tax_unit, period, ["taxable_social_security"])
```

Reply:
Nick @nmrodelo,  Can you please make the stylistic change requested by @MaxGhenis, so that PR #1780 can be merged?